RELEASES.md

```diff
@@ -1,3 +1,61 @@
+Version 1.21.0 (2017-10-12)
+==========================
+
+Language
+--------
+- [Relaxed path syntax. You can now add type parameters to values][43540]
+  Example: `struct Foo<T>(T); let bar = Foo::<u8>(0);`.
+
+Compiler
+--------
+- [Upgraded jemalloc to 4.5.0][43911]
+- [Enabled unwinding panics on Redox][43917]
+
+Libraries
+---------
+- [Generate builtin impls for `Clone` for `[T: N]` where `N` is between 0
+  and 32][43690]
+- [`Stdin`, `Stdout`, and `Stderr` now implement `AsRawFd`.][43459]
+- [`Rc` and `Arc` now implement `From<&[T]> where T: Clone`, `From<str>`,
+  `From<String>`, `From<Box<T>> where T: ?Sized`, and `From<Vec<T>>`.][42565]
+
+Stabilized APIs
+---------------
+
+Cargo
+-----
+- [You can now call `cargo install` with multiple package names][cargo/4216]
+- [Cargo commands inside a virtual workspace will now implicitly
+  pass `--all`][cargo/4335]
+
+Misc
+----
+- [Cargo docs are moving][43916]
+  to [doc.rust-lang.org/cargo](//doc.rust-lang.org/cargo)
+- [The rustdoc book is now available][43863]
+  at [doc.rust-lang.org/rustdoc](//doc.rust-lang.org/rustdoc)
+
+Compatibility Notes
+-------------------
+- [Remove the trait selection impl in method::probe][43880] This may cause
+  breakage in subtyping corner cases.
+- [make JSON error's byte position start at top of file.][42973] Was previously
+  relative to the `CodeMap` which required unstable code.
```

Comment:
Release note readers shouldn't need to think about named internal implementation details like `CodeMap`, and I'm not sure where "which required unstable code" is coming from.

Maybe something like "was previously relative to a concatenation of files in a crate"?? (I don't actually know how the compiler determines which files go in a `CodeMap`—and neither does typical consumer of error-message JSON, which is why we fixed this).